src/main/java/com/github/kongchen/swagger/docgen/mavenplugin/SecurityDefinition.java

```diff
@@ -55,8 +58,8 @@
             while (securityDefinitionNameIterator.hasNext()) {
                 String securityDefinitionName = securityDefinitionNameIterator.next();
                 JsonNode securityDefinition = tree.get(securityDefinitionName);
-                securityDefinition = ((ObjectNode) securityDefinition).put("name", securityDefinitionName);
-                securityDefinitions.add(securityDefinition);
+        //        securityDefinition = ((ObjectNode) securityDefinition).put("name", securityDefinitionName);
```

Comment:
If this line is no longer needed, please delete it instead of just commenting it out.